### PR TITLE
Java 11 Read Me updates

### PR DESCRIPTION
**Pull request by averikitsch**

### CODE REVIEW

File: appengine-java11/README.md [hunk 1]
```diff
@@ -48,6 +48,23 @@ To switch to an Open JDK 11 in a Cloud shell session, you can use:
 
 ## Java 11 runtime
 
+The simplest way to deploy to App Engine Java 11 is using an executable [Uber JAR][uber-jar]. App Engine will automatically configure the `entrypoint` to run the JAR file.
```
Comment (lesv) [inline appengine-java11/README.md]: is `[...][...]` correct here?

File: appengine-java11/appengine-simple-jetty-main/README.md [hunk 2]
```diff
@@ -70,7 +70,7 @@ the provided server is running your application as expected.
 
 - Start the server with your `WAR` file as an argument:
 ```
-mvn exec:java -Dexec.args="<path/to/your/war/file"
+mvn exec:java -Dexec.args="path/to/your/war/file"
```
Comment (lesv) [inline appengine-java11/appengine-simple-jetty-main/README.md]: `-Dexec.args="target/path.war"` perhaps or something that mentions target

File: appengine-java11/README.md (hunk 1 above)
Comment (averikitsch) [reply appengine-java11/README.md]: Yes, when using a variable.

File: appengine-java11/appengine-simple-jetty-main/README.md (hunk 2 above)
Comment (averikitsch) [reply appengine-java11/appengine-simple-jetty-main/README.md]: done.

**Pull request merged**